app/services/trust_update_service.rb

```diff
@@ -15,6 +18,13 @@ def update_trusts(last_update: nil)
 
 private
 
+  def create_trusts
+    existing_trust_ids = Trust.pluck(:companies_house_number)
+    DataStage::Trust.where.not(companies_house_number: existing_trust_ids).find_each do |staged_trust|
```

Comment:
I think we might need to do DataStage::Trust.gias_status_open, there is no point importing in closed trusts.